apps/src/multicore_server.rs

```diff
@@ -0,0 +1,785 @@
+                        },
+            
+                        quiche::PathEvent::ReusedSourceConnectionId(cid_seq, old, new) => {
+                            info!(
+                                "{} Peer reused cid seq {} (initially {:?}) on {:?}",
+                                client.conn.trace_id(),
+                                cid_seq,
+                                old,
+                                new
+                            );
+                        },
+            
+                        quiche::PathEvent::PeerMigrated(local_addr, peer_addr) => {
+                            info!(
+                                "{} Connection migrated to ({}, {})",
+                                client.conn.trace_id(),
+                                local_addr,
+                                peer_addr
+                            );
+                            scheduler.add_path((local_addr, peer_addr)) // TODO check if valid
+                        },
+            
+                        quiche::PathEvent::PeerPathStatus(addr, path_status) => {
+                            info!("Peer asks status {:?} for {:?}", path_status, addr,);
+                            client
+                                .conn_paths
+                                .set_path_status( addr.0, addr.1, path_status, false)
+                                .map_err(|e| error!("cannot follow status request: {}", e))
+                                .ok();
+                        },
+
+                        quiche::PathEvent::PacketNumSpaceDiscarded((local, peer), epoch, handshake_status, now) => {
+                            let p = client.conn_paths.get_mut_from_addr(local, peer).unwrap();
+                            p.recovery.on_pkt_num_space_discarded(epoch, handshake_status, now);
+                        },
+                    }
+                }, 
+                None => break,
+            }
+        }
+    }
+}
+
+/// Set SO_REUSEPORT socket option.
+///
+/// Note that this socket option is set only on linux platforms.
+#[cfg(target_os = "linux")]
+fn set_so_reuseport_sockopt(sock: &mio::net::UdpSocket) -> io::Result<()> {
+    use nix::sys::socket::setsockopt;
+    use nix::sys::socket::sockopt::ReusePort;
+    use std::os::unix::io::AsRawFd;
+
+    // mio::net::UdpSocket doesn't implement AsFd (yet?).
+    let fd = unsafe { std::os::fd::BorrowedFd::borrow_raw(sock.as_raw_fd()) };
+
+    setsockopt(&fd, ReusePort, &true)?;
+    
+    Ok(())
+}
+*/
+
+pub fn multicore_start_server(args: ServerArgs, conn_args: CommonArgs) {
+    env_logger::builder()
+        .default_format_timestamp_nanos(true)
+        .init();
+
+    let mut pacing = false;
+
+    // Dumy socket used for detecting gso 
+    let socket = mio::net::UdpSocket::bind("127.0.0.1:0".parse().unwrap()).unwrap(); 
+
+    // Set SO_TXTIME socket option on the listening UDP socket for pacing
+    // outgoing packets.
+    if !args.disable_pacing {
```

Comment:
You should add to your evaluation script the configuration of a fq 

`$ tc qdisc add dev itf_name root fq flow_limit 10000 limit 100000`